### PR TITLE
fix: Unreferenced Microsoft.Insights files

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: specification/applicationinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -274,6 +274,7 @@ These settings apply only when `--tag=package-2018-06-17-preview` is specified o
 
 ``` yaml $(tag) == 'package-2018-06-17-preview'
 input-file:
+- Microsoft.Insights/preview/2018-06-17-preview/workbookOperations_API.json
```
Comment (veronicagg) [inline specification/applicationinsights/resource-manager/readme.md]: This is up to the owner team to include, depending on what they want to include in the specific package. Tagging @ericc1103 who introduced this API version to check whether this should be included here or not.

**Review by veronicagg — CHANGES_REQUESTED** [review]
@nschonni The package tags don't necessarily need to contain every file for the version, it's up to each team to include them. We recently added a tool to check on not referenced files in CI, but haven't turned it on as requiredcand this is one of the reasons. Examples would be ok to delete but we should be careful adding files to packages.

File: specification/applicationinsights/resource-manager/readme.md (hunk 1 above)
Comment (ericc1103) [reply specification/applicationinsights/resource-manager/readme.md]: Adding @jgardner04 to discuss this together.

**Comment by veronicagg** [comment]
@nschonni @ericc1103 @jgardner04 I haven't seen updates on this PR for more than 10 days, so I'm closing it, please reopen or tag me when you get back to it. thanks!

**Comment by nschonni** [comment]
@veronicagg this is something flagged by Avocado, but I'll leave this to you to correct internally then

**Pull request closed without merging**